src/Parser/Hunter/Marksmanship/Modules/Features/TimeFocusCapped.js

```diff
@@ -21,36 +25,50 @@ class TimeFocusCapped extends Module {
+        label='Time Focus Capped'
+        tooltip={`You wasted <b> ${totalFocusWaste}  </b> focus. <br />
+        That's <b>  ${formatPercentage(totalFocusWaste / (this.owner.fightDuration / 1000 * this.focusTracker.focusGen))}% </b> of your total focus generated.
         <br /> For more details, see the Focus Chart tab.`}
-        value =  {`${percentCapped} %`}
-      //Time not Focus-Capped: {Math.round((this.owner.fightDuration / 1000 - this.focusTracker.secondsCapped) * 100) / 100}s / {Math.floor(this.owner.fightDuration / 1000)}
-      footer={(
-        <div className="statistic-bar">
-        <div
-          className="stat-health-bg"
-          style={{ width: `${(100-percentCapped)}%` }}
-          data-tip={`You spent <b>${100-percentCapped}%</b> of your time, or <b>${Math.round(Math.floor(this.owner.fightDuration/1000) - this.focusTracker.secondsCapped)}s</b> under the focus cap.`}
-        >
-        </div>
-          <div
-            className="stat-overhealing-bg"
-            style={{ width: `${percentCapped}%` }}
-            data-tip={`You spent <b>${percentCapped}%</b>, or <b>${Math.round(this.focusTracker.secondsCapped)}s</b> of your time focus capped.`}
-          >
+        value={`${percentCapped} %`}
+        //Time not Focus-Capped: {Math.round((this.owner.fightDuration / 1000 - this.focusTracker.secondsCapped) * 100) / 100}s / {Math.floor(this.owner.fightDuration / 1000)}
+        footer={(
+          <div className="statistic-bar">
+            <div
+              className="stat-health-bg"
+              style={{ width: `${(100 - percentCapped)}%` }}
+              data-tip={`You spent <b>${100 - percentCapped}%</b> of your time, or <b>${Math.round(Math.floor(this.owner.fightDuration / 1000) - this.focusTracker.secondsCapped)}s</b> under the focus cap.`}
+            >
+            </div>
+            <div
+              className="stat-overhealing-bg"
+              style={{ width: `${percentCapped}%` }}
+              data-tip={`You spent <b>${percentCapped}%</b>, or <b>${Math.round(this.focusTracker.secondsCapped)}s</b> of your time focus capped.`}
+            >
+            </div>
           </div>
-        </div>
-      )}
-      footerStyle={{ overflow: 'hidden' }}
+        )}
+        footerStyle={{ overflow: 'hidden' }}
       />
     );
   }
+  suggestions(when) {
+    const totalFocusWaste = this.getTotalWaste;
+    const percentCapped = formatPercentage(this.focusTracker.secondsCapped / (this.owner.fightDuration / 1000));
+    
+    when(percentCapped).isGreaterThan(2.5)
+      .addSuggestion((suggest, actual, recommended) => {
+        return suggest(<span>You're spending a long time focus capping, try and avoid this as it is a significant DPS loss. It's better to shoot a non-vulnerable <SpellLink id={SPELLS.AIMED_SHOT.id} />, than spend time at max focus. You wasted a total of {totalFocusWaste} focus over the course of the fight.</span>)
```

Comment:
"You're spending a lot of time being focus capped, ty to avoid this.." sounds more natural to me.